packages/jellyfish-transaction/src/script/dftx/dftx.ts

```diff
@@ -91,7 +91,9 @@ import {
   CTakeLoan,
   TakeLoan,
   CPaybackLoan,
-  PaybackLoan
+  PaybackLoan,
+  CloseVault,
+  CCloseVault
```

Comment:
better move to vault section

Reply:
This is the result of merge conflict. I am going to fix this in another PR.

Reply:
Fixed in updateVault DFTX PR